squad-server/plugins/discord-admin-request.js

```diff
@@ -123,9 +131,39 @@ export default class DiscordAdminRequest extends DiscordBasePlugin {
 
     await this.sendDiscordMessage(message);
 
-    await this.server.rcon.warn(
-      info.player.steamID,
-      `An admin has been notified, please wait for us to get back to you.`
-    );
+    await this.server.updatePlayerList;
+    let admins = await this.server.getAdminsWithPermission('canseeadminchat');
+
+    if (this.options.warnInGameAdmins) {
+      for (const player of this.server.players) {
+        if (admins.includes(player.steamID)) {
+          this.amountAdmins++;
```

Comment:
This can be local to the function. Add before this loop `let adminAmount = 0;` and then replace this with `adminAmount++;`.

You can then remove the `this.adminAmount = 0;` below.